tasks/install/Debian.yml

```diff
@@ -20,10 +20,17 @@
   notify: filebeat restart
   when: filebeat_use_apt_repo
 
-- name: Install FileBeat from URL
+- name: Download FileBeat .deb
+  get_url:
+    src: "{{ filebeat_deb_baseurl }}/{{ item }}-{{ filebeat_version }}-amd64.deb"
```

Comment:
The parameter here should actually be `url`. I'll merge the change and commit a fix. I'd recommend to run the vagrant tests before submitting the PRs.